reports-ui/src/main/java/io/jmix/reportsui/screen/report/run/ReportRun.java

```diff
@@ -96,7 +96,7 @@ public void setScreenParameter(String screenParameter) {
     }
 
     @Subscribe
-    protected void onInit(InitEvent event) {
+    protected void onBeforeShow(BeforeShowEvent event) {
```

Comment:
`onInit` is called before setting up screenParameter from setter and we get `null`